README.md

```diff
@@ -44,7 +44,7 @@ makepkg -si
 
 Contributors are always welcome!
 
-If you plan to make any large changes or changes that may not be 100% agreed
+If you plan to make larger changes or changes that may not be 100% agreed
```

Comment:
I prefer the old wording. Is there anything wrong with it?

Reply:
The combination of "any" with the plural "changes" sounds a bit weird to some people, even though it's not technically incorrect as far as I know. I believe this change makes it sound a bit more natural (native speakers: please correct me here).

Reply:
Well larger means bigger than something. So to me when you say larger without comparing it to anything it does not quite make sense.

Reply:
Then we should probably remove the word "large" altogether, as you can't really quantify code changes that way (where does large begin?). How about "If you plan to make more involved/complex/elaborate/intricate changes..."?

Reply:
You could say the same thing about all of these. What makes a change complex? Yes large is subjective but that's fine. And a change does not have to be complex to merit an issue anyway. You could be tearing out a large piece of the code base in order to make it less complex. Basically what #585 is doing right now.

Reply:
@polygamma you're stalking the repo right now. Got any opinions with your German-English?

Reply:
my 2 cents: `If you plan to make any large changes` is fine

Reply:
As mentioned I'm not a native speaker either, but I have spend half my life in english speaking countries. The word 'any' with an innumerable plural object simply sounds a bit awkward.

A 'large change' by definition stands in contrast to a smaller change, hence why I would use the comparative "larger" here.

"Edits from maintainers" are enabled, so you do you here.

Reply:
Both of them sound fine. Personally I prefer `any large changes` even if it's more Americanised

Reply:
@muesli There's three of us in agreement on keeping it. It would be much easier if you edited your own commits than asking us to do it.